replication-api/src/main/java/org/codice/ditto/replication/api/data/ReplicationSite.java

```diff
@@ -11,31 +11,36 @@
  * License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package org.codice.ditto.replication.api.modern;
-
-import java.net.URL;
+package org.codice.ditto.replication.api.data;
 
 /** A ReplicationSite holds information about a system to be replicated to/from */
-public interface ReplicationSite {
+public interface ReplicationSite extends Persistable {
 
   /**
-   * Get the unique ID for this site
+   * Get the human readable name of this site
    *
-   * @return site ID
+   * @return site name
    */
-  String getId();
+  String getName();
 
   /**
-   * Get the human readable name of this site
+   * Set the name of this site
    *
-   * @return site name
+   * @param name the name to give this site
    */
-  String getName();
+  void setName(String name);
```

Comment:
✏️  Since SiteLoader has a method `saveSite(String name, String url)`, these setters and the `createPersistable(Class<T> klass)` seem to duplicate the same behavior.